tests/s3/test_bucket_location.py

```diff
@@ -125,24 +126,22 @@ def test_get_bkt_loc_valid_bkt_272(self):
     @CTFailOn(error_handler)
     def test_get_bkt_loc_bkt_not_present_273(self):
         """verify get bucket location for the bucket which is not present."""
-        self.log.info(
-            "Verify get bucket location for the bucket which is not present")
-        self.log.info(
-            "Step 1 : Check the bucket location on non existing bucket %s ",
+        self.log.info("STARTED: Verify get bucket location for the bucket which is "
```

Comment:
Indentation of the original code needs to be restored here for lines 129-131

Reply:
In past review, I got review comment to restore such kind of indentation. I am unsure what to follow exactly.